app/views/restaurants/show.html.haml

```diff
@@ -7,13 +7,14 @@
 .br
 .btn
   %p
-  = link_to @restaurant.name, root_path, class: 'btn btn-primary'
+  = link_to 'Back', root_path, class: 'btn btn-primary'
```

Comment:
we have a link home that goes  to root path, shouldnt back go to last page you were on?

Reply:
Same thing here I guess. I could add a back-button instead if you want :)

Reply:
just think its redundant to have a back button to root path since we have a home button to root path.
could try `request.referer` to go to last page user was on.

Reply:
I misunderstood you, haha